src/runtime/go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/kata-containers/kata-containers/src/runtime
 
 go 1.21
+toolchain go1.24.1
```

Comment:
@stevenhorsman I forgot what we discussed - is this ok? Maybe I'll start a short doc on guidance for reviewing these dependabot PRs 🙂

Reply:
So if not sure the best practice, but I'm not sure if we even need the toolchain. What we definitely don't want it a 1.24 toolchain, which shouldn't be necessary after we make the 1.23 bump land as this module only needs 1.23.0+ https://cs.opensource.google/go/x/net/+/refs/tags/v0.38.0:go.mod. I'm not quite sure why dependabot is moving it to 1.24.1, but maybe when it's not got a currently compatible version then it just picks the latest?